src/components/wallet/methods/withdrawAsset.ts

```diff
@@ -113,82 +88,123 @@ export default async function withdrawAsset(
+              .accounts()
+              .accountId(keypair.publicKey())
+              .call()
+            this.logger.response('Fetching account sequence number', sequence)
+            const account = new Account(keypair.publicKey(), sequence)
+            const txn = new TransactionBuilder(account, {
+              fee: BASE_FEE,
+              networkPassphrase: this.network_passphrase,
             })
-          )
-          .addMemo(memo)
-          .setTimeout(0)
-          .build()
-
-        txn.sign(keypair)
-        await this.server.submitTransaction(txn)
-        transactionSubmitted = true
-        console.log('just submitted transaction')
-
-        const urlBuilder = new URL(transaction.more_info_url)
-        urlBuilder.searchParams.set('callback', 'postMessage')
-        popup.location.href = urlBuilder.toString()
-      } else {
-        setTimeout(() => {
-          const urlBuilder = new URL(transaction.more_info_url)
-          urlBuilder.searchParams.set('callback', 'postMessage')
-          popup.location.href = urlBuilder.toString()
-        }, 1000)
+              .addOperation(
+                Operation.payment({
+                  destination:
+                    transactionJson.transaction.withdraw_anchor_account,
+                  asset: new Asset(assetCode, assetIssuer),
+                  amount: transactionJson.transaction.amount_in,
+                })
+              )
+              .addMemo(memo)
+              .setTimeout(0)
+              .build()
+
+            txn.sign(keypair)
+            this.logger.request(
+              'Submitting withdrawal transaction to Stellar',
+              txn
+            )
+            const horizonResponse = await this.server.submitTransaction(txn)
+            this.logger.response(
+              'Submitting withdrawal transaction to Stellar',
+              horizonResponse
+            )
+            break
+          }
+          case 'pending_anchor': {
+            this.logger.instruction('The anchor is processing the transaction')
+            break
+          }
+          case 'pending_stellar': {
+            this.logger.instruction(
+              'The Stellar network is processing the transaction'
+            )
+            break
+          }
+          case 'pending_external': {
+            this.logger.instruction(
+              'The transaction is being processed by an external system'
+            )
+            break
+          }
+          case 'pending_user': {
+            this.logger.instruction(
+              'The anchor is waiting for you to take the action described in the popup'
+            )
+            break
+          }
+          case 'error': {
+            this.logger.instruction(
+              'There was a problem processing your transaction'
+            )
+            break
+          }
+        }
       }
+      // run loop every 2 seconds
+      await new Promise((resolve) => setTimeout(resolve, 2000))
     }
+
```

Comment:
We should include the same  logger instruction here in withdraw asset
```suggestion
    this.logger.instruction(`Transaction status: ${currentStatus}`)
    if (!['completed', 'error'].includes(currentStatus) && popup.closed) {
      this.logger.instruction(
        'The popup was closed before the transaction reached a terminal status, ' +
          'if your balance is not updated soon, the transaction may have failed.'
      )
    }
```